src/documents/LocalizationInThePreferencesFramework.md

```diff
@@ -130,9 +130,9 @@ Any panel that has the grade `fluid.prefs.defaultPanel` will have access to the
 ```javascript
 fluid.defaults("my.messageLoader", {
```

Comment:
Shall we remove the example of creating message loader too? As in the above comment, if the grade version of preferences framework was no longer recommend to integrators, integrators would not need to create their own message loaders any more.